src/msg/async/Stack.cc

```diff
@@ -68,6 +68,9 @@ std::shared_ptr<NetworkStack> NetworkStack::create(CephContext *c, const string
     return std::make_shared<DPDKStack>(c, t);
 #endif
 
+  lderr(c) << __func__ << " ms_async_transport_type " << t <<
+    " is not valid in this version of ceph" << dendl;
```

Comment:
maybe not related to version because built without ENABLE_XXX, " xxx isn't supported?"